### ISSUE TITLE
TODO: Create tests/ when creating module

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [Abdur-rahmaanJ/shopcube#267]
Create tests folder when creating module. Code found in shopyoapi/cmd.py - create_module

**Comment by damiaanh** [comment]
Hi, I'm new to Github and would to like contribute by addressing this issue. Where should such a folder be created? If still needed, could you assign me to this issue?

**Comment by rehmanis** [comment]
Hi, thank you for the interest. If you haven't setup the repo you can follow [this guide](https://www.appinv.science/shopyo/contrib.html#setup-instructions) to help you set it up.

For this app we have written an API that allows user to auto create a module with some preexisting contents. The user has the option to create a module inside a box by running the commands as mentioned in the [docs here](https://www.appinv.science/shopyo/modules.html#boxes-or-many-apps-together) (`python manage.py startsubapp demoapp in box__demo`) where `box__demo` is the root module inside which we create a submodule `demoapp`. For testing you can also just run the command as mentioned in the [doc here](https://www.appinv.science/shopyo/modules.html#easy-way-to-create-module)(`python manage.py startapp modulename`) to create a module `modulename` with some preexisting content.

You need to update the `create_module`  function (inside `shopyo/shopyoapi/cmd.py`) which currently creates the following preexisting content if you run `python manage.py startapp modulename`.  
```bash
modulename/
├── forms.py
├── global.py
├── info.json
├── models.py
├── templates
│   └── modulename
│       ├── blocks
│       │   └── sidebar.html
│       └── dashboard.html
└── view.py
```

We want to add a `tests` folder inside `modulename/` on the same level as `forms.py`. The `tests` folder should contain the files `test_<modulename>_functional.py` and `test_<modulename>_models.py`. As for the contents for these files, you can add some comments such as "Please add your tests to this file". I still need to come up with some pretests for these files

Once you are ready to start working check the doc guide for [pull request](https://www.appinv.science/shopyo/contrib.html#making-a-pull-request)

**Comment by Abdur-rahmaanJ** [comment]
Join our [discord server](https://discord.gg/k37Ef6w) for more help ^^!

**Comment by rehmanis** [comment]
Closed by #291
